File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    l = list(s)
    
    cnt = 0;
    for i in range(len(l)):
        if (l[i] == ' '):
            answer += " "
            cnt = 0
            continue
        
        if (cnt & 1):
            answer += l[i].lower()
        else:
            answer += l[i].upper()
            
        cnt += 1
    return answer

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>
#include <cctype>

using namespace std;

string solution(string s) {
    string answer = "";
    
    bool isOdd = false;
    for (int i = 0; i < s.size(); i++) {
        if (s[i] == ' ') {
            isOdd = false;
            continue;
        }
        
        if (isOdd) {
            isOdd = false;
            s[i] = tolower(s[i]);
        } else {
            isOdd = true;
            s[i] = toupper(s[i]);
        }
    }
    return s;
}